tutorials/math/matrices_and_transforms.rst

```diff
@@ -308,36 +308,25 @@ as well as :ref:`Spatial.to_local() <class_Spatial_method_to_local>` and :ref:`S
 Scale
 ~~~~~
 
-A matrix can be scaled too. Scaling will multiply the basis vectors by a
-vector (X vector by x component of the scale, Y vector by y component of
-the scale). It will leave the origin alone:
+A matrix can be scaled too. Scaling will multiply the X and Y vectors of the
+matrix by the ``x`` and ``y`` components of another vector (``X * x``, ``Y * y``).
+It will leave the origin alone:
 
 .. tabs::
  .. code-tab:: gdscript GDScript
 
-    # Make the basis twice its size.
+    # Scale the transform's vectors to twice their size
     var m = Transform2D()
     m = m.scaled( Vector2(2,2) )
 
  .. code-tab:: csharp
 
-    // Make the basis twice its size.
+    // Scale the transform's vectors to twice their size
     var m = Transform2D.Identity;
     m = m.Scaled(new Vector2(2, 2));
 
 .. image:: img/tutomat15.png
 
-These kind of operations in matrices are accumulative. It means every
-one starts relative to the previous one. For those who have been living
-on this planet long enough, a good reference of how transform works is
-this:
-
-.. image:: img/tutomat16.png
```

Comment:
The image should be removed from the repository, so we don't leave an unused image around :slightly_smiling_face:

Reply:
Good point.